tests/test_basic_cluster.nim

```diff
@@ -0,0 +1,41 @@
+# nim-raft
+# Copyright (c) 2023 Status Research & Development GmbH
+# Licensed under either of
+#  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+#  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+# at your option.
+# This file may not be copied, modified, or distributed except according to
+# those terms.
+
+import unittest2
```

Comment:
These tests looks like simulations and as actual tests. No acceptance criteria is defined (i.e. what the conditions that would make the test fail?)

Reply:
I am not done with this test yet.